GeneratorInterface/LHEInterface/plugins/LHEWriter.cc

```diff
@@ -43,8 +46,12 @@ void LHEWriter::beginRun(const edm::Run &run, const edm::EventSetup &es)
 {
 	edm::Handle<LHERunInfoProduct> product;
 	run.getByLabel("source", product);
+        
+        if (!product.isValid()) {
+          run.getByLabel("externalLHEProducer", product);
```

Comment:
(Sorry, missed this before).
This being 53x I think not.

I'll make sure it gets added when this is put in the newer releases.